omnigibson/scenes/scene_base.py

```diff
@@ -59,20 +59,35 @@ def __init__(
         self,
         scene_file=None,
         use_floor_plane=True,
+        floor_plane_size=1000.0,
         floor_plane_visible=True,
         floor_plane_color=(1.0, 1.0, 1.0),
+        floor_plane_grid=False,
+        floor_plane_grid_resolutions=(0.001, 0.01, 0.1, 1.0),
         use_skybox=True,
+        use_skybox_texture=True,
```

Comment:
hmm I am not fan of a variable named `use_skybox_texture` can be True/False/an string. Can we create another variable for the string case?

Reply:
I think for now it's fine to let the texture path come from a macro rather than a config str, it's not quite important to change from a config. So I'm changing this to just True/False.